milestone2.ts

```diff
@@ -83,34 +83,75 @@ const userData: Volunteer[] = [
   Hint: user data is stored in the userData object above. 
 */
 type GetNumber = (data: Volunteer[]) => number;
-let findAverage: GetNumber; // Define lambda function here
+let findAverage: GetNumber = (data)=>{
+  var total = 0;
+  for(var i=0; i<data.length;i++)
```

Comment:
[Use let, not var](https://stackoverflow.com/questions/762011/what-is-the-difference-between-let-and-var)